_posts/2018-10-15-arms-proposal.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "ARMS CSC 596 Proposal"
+categories: ["Missouri State University", "ARCS"]
+tags: ["self-study", "csc596", "robotics", "python"]
+author: "Dillon Flohr"
+---
+
+# CSC 596 Proposal
```

Comment:
Combine this with the h3 below. It should be the same as the title in yaml at the top of the file.

# ARMS CSC 596 Proposal